### PR TITLE
Display messages from previous exceptions

### PR DESCRIPTION
**Pull request by florianorben**
Hey!

This will show a list of all the previous exceptions that have been thrown before the current exception as described in #593.

It'll look like this
preview

References #593 .

### CODE REVIEW

File: src/Whoops/Resources/views/header.html.php [hunk 1]
```diff
@@ -15,6 +15,25 @@
...
+
+      <?php if (count($previousMessages)): ?>
+        <div class="exc-title prev-exc-title">
+          <span class="exc-title-secondary">Previous Exceptions</span>
+        </div>
+
+        <ul>
+          <?php foreach ($previousMessages as $i => $previousMessage): ?>
+            <li>
+              <span class="prev-exc-code">(<?php echo $previousCodes[$i] ?>)</span>
```
Comment (staabm) [inline src/Whoops/Resources/views/header.html.php]: In the top-level exception we used to have the exc.-behind the message

Comment (florianorben) [reply src/Whoops/Resources/views/header.html.php]: Oops, your're right. :)
image
Changed now

**Comment by staabm** [comment]
Really like this. Didnt have time to test it

**Comment by florianorben** [comment]
Also, changed "Previous Exceptions" to "Previous exceptions" (lowercase "e").
Didn't update the screenshot though :)

**Comment by florianorben** [comment]
@denis-sokolov Done! Was used to using typehints, but PHP 5.6 bugged with using  `\Throwable` :)
The latest HHVM build seems to have bugged because of some travis issue
```
OCI runtime exec failed: exec failed: container_linux.go:348: starting container process caused "process_linux.go:86: executing setns process caused \"exit status 21\"": unknown
```
is there any way to trigger a new build without a push?

**Comment by staabm** [comment]
I guess open+close PR will work

**Comment by florianorben** [comment]
Didn't work sadly :( Changing a random comment and a new push did it.
All builds green now :)

**Comment by denis-sokolov** [comment]
CIs usually have a button to restart the build. In this case only I have access to it if I am logged in.

Re-pushing works in all cases, though. I usually do a `git commit --amend`, don’t change the commit message, and `git push -f`. An amend updates the timestamp which modifies the hash which makes it a new commit.

Thank you for a great contribution!

**Pull request merged**